requirements/dev.in

```diff
@@ -52,4 +52,7 @@ python-digitalocean==1.14.0
 # Needed for updating the locked pip dependencies
 pip-tools==2.0.2
 
+# zulip's linting framework - zulint
+-e git+https://github.com/zulip/zulint@master#egg=zulint
```

Comment:
We generally put in a fake version in these GitHub lines -- you can do 0.0.1 here.